dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/TransformToolBox.java

```diff
@@ -0,0 +1,183 @@
+        this.fileAssetAPI = fileAssetAPI;
+        this.versionableAPI = versionableAPI;
+        this.userAPI = userAPI;
+        this.contentletAPI = contentletAPI;
+        this.htmlPageAssetAPI = htmlPageAssetAPI;
+        this.contentHelper = contentHelper;
+    }
+
+    public TransformToolBox() {
+        this(APILocator.getIdentifierAPI(), APILocator.getHostAPI(), APILocator.getLanguageAPI(),
+                APILocator.getFileAssetAPI(), APILocator.getVersionableAPI(),
+                APILocator.getUserAPI(), APILocator.getContentletAPI(),
+                APILocator.getHTMLPageAssetAPI(), ContentHelper.getInstance());
+    }
+
+    public static Contentlet copyContentlet(final Contentlet contentlet) {
+        final Contentlet newContentlet = new Contentlet();
+        if (null != contentlet && null != contentlet.getMap()) {
+            newContentlet.getMap().putAll(contentlet.getMap());
+        }
+        return newContentlet;
+    }
+
+    static Map<String, Object> mapLanguage(final Language language, final boolean wrapAsMap) {
+
+        final Builder<String, Object> builder = new Builder<>();
+
+        builder
+                .put("languageId", language.getId())
+                .put("language", language.getLanguage())
+                .put("languageCode", language.getLanguageCode())
+                .put("country", language.getCountry())
+                .put("countryCode", language.getCountryCode())
+                .put("languageFlag", getLiteralLocale(language.getLanguageCode(), language.getCountryCode()));
+
+        final String iso = UtilMethods.isSet(language.getCountryCode())
+                ? language.getLanguageCode() + StringPool.DASH + language.getCountryCode()
+                : language.getLanguageCode();
+        builder.put("isoCode", iso.toLowerCase());
+
+        if(wrapAsMap){
+            builder.put("id", language.getId());
+            return ImmutableMap.of("languageMap", builder.build(), "language",language.getLanguage());
+        }
+        return builder.build();
+    }
+
+    static Map<String, Object> mapIdentifier(final Identifier identifier, final boolean wrapAsMap) {
+
+        final Builder<String, Object> builder = new Builder<>();
+        builder
+                .put("identifier", identifier.getId())
+                .put("parentPath", identifier.getParentPath())
+                .put("hostId", identifier.getHostId());
+
+        if(wrapAsMap){
+            builder.put("id", identifier.getId());
+            return ImmutableMap.of("identifierMap", builder.build(), "identifier",identifier.getId());
+        }
+        return builder.build();
+    }
+
+    public static Map<String, Object> transform(final Field field, final Contentlet contentlet) {
+        File file;
+        try {
+            file = contentlet.getBinary(field.variable());
+        } catch (IOException e) {
+            throw new DotStateException(e);
+        }
+
+        if(file==null) return Collections.emptyMap();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)